features.md

```diff
@@ -17,7 +17,7 @@ Test suites, test cases, directory names… feel free to rename everything to fi
 <section class="visual-section">
 ## Execution engines
 
-By default, atoum provides three execution engines:
+By default, atoum provides three <a href="http://docs.atoum.org/en/latest/written_help.html#execution-engine">execution engines</a>:
```

Comment:
because I don't remember the syntax and the first link I ahve seen was a "<a" link